modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -252,6 +252,23 @@ protected function register_controls() {
 		);
 		$this->end_controls_section();
 
+		$this->start_controls_section( 'section_interactions', [
+			'label' => esc_html__( 'Interactions section', 'elementor' ),
+		] );
+
+		$this->add_control( 'default_state', [
+			'label'              => esc_html__( 'Default State', 'elementor' ),
+			'type'               => Controls_Manager::SELECT,
+			'options'            => [
+				'expanded' => esc_html__( 'First expended', 'elementor' ),
```

Comment:
The spelling of expanded/expended is different from each other. Is that on purpose? I think it should be 'expanded' in both cases.